handbook/ops/roles/director_business_operations.md

```diff
@@ -0,0 +1,66 @@
+# Director of Business Operations
+
+Sourcegraph's business operations team empowers our product, design, and engineering organizations with the insights, tools, and data they need to make better decisions. As the Director of Business Operations, you will build the team that helps our product organization scale and serve every developer in the world.
+
+You will own product analytics, data engineering and infrastructure, project management and internal consulting, and will be a key voice in setting company strategy.
+
+Our team and our sales are growing fast—we’ve proven strong product-market fit, with wall-to-wall usage and top-tier engagement, genuine love from our users and customers, and tens of thousands of paid daily active users at some of the world’s premier development teams.
+
+## Responsibilities
+
+### In the first 30 days you will...
+
+- Conduct a comprehensive review of our business operations, including the metrics and goals we track, the data we collect, to systems we use, and the teams and projects we serve, and build a team growth roadmap.
+  - Conduct interviews with the entire executive team to understand gaps and needs.
+- Work closely with and get to know our existing business operations and broader ops team.
+- Go through an intensive training program on our product, our value propositions, our customers, our team, and our processes.
+- Take ownership of our internal and external reporting and our business operations team goals.
+
+
+### In the first 60 days you will…
+
+- Begin to build a high-agency business operations team, likely including new hires such as:
```

Comment:
Do you expect this hire to settle in before hiring for the other positions, or we'll hire the positions concurrently?

Reply:
Concurrently (for certain roles). This is written with real-world, current needs, and if it changes, we would update.